utils/format.go

```diff
@@ -836,6 +876,27 @@ func FormatValidatorWithName(validator interface{}, name string) template.HTML {
 	}
 }
 
+func FormatValidatorWithNameAlt(validator interface{}, name string) template.HTML {
+	var validatorRead string
+	var validatorLink string
+	switch v := validator.(type) {
+	case []byte:
+		if len(v) > 2 {
+			validatorRead = fmt.Sprintf("0x%x…%x", v[:2], v[len(v)-2:])
```

Comment:
If `len(v)` is <= 2, validatorRead is not set. This is not intended, is it?

Reply:
This function (which I have deleted since) was basically a copy of the one above. I can't tell for sure if this is intended as the code was written by somebody not working for bitfly anymore. Also I doubt whether the validator parameter is ever passed as a byte array. I will leave this be as is for now and probably replace this function over the course of implementing all new designs.